### PR TITLE
Fix print download on chrome when using inlinePrintOutput

### PR DESCRIPTION
**Pull request by cazitouni**
Should fix https://github.com/qgis/qwc2-demo-app/issues/504.
Didn't find solution for the filename but, since you get the correct .pdf and you save it you can change the name anyway so maybe it's not that bad.

Let me know if it works for you.
I've tested to some extent with the demo project on firefox and chrome but not with atlas.

Kind regards,
Clement.

### CODE REVIEW

File: plugins/Print.jsx [hunk 1]
```diff
@@ -545,7 +544,7 @@ class Print extends React.Component {
     print = (ev) => {
         if (this.props.inlinePrintOutput) {
             this.setState({printOutputVisible: true, outputLoaded: false});
-        } else {
+        }
```
Comment (manisandro) [inline plugins/Print.jsx:547]: Can you fix the indentation of the following lines?

Comment (manisandro) [reply plugins/Print.jsx]: The subsequent lines should be de-indented as the indentation block was removed.

Comment (cazitouni) [reply plugins/Print.jsx:547]: MB sorry, should be fixed

**Comment by manisandro** [comment]
Thanks!

**Pull request merged**